server/modules/analysis/manager/processingChainManager.js

```diff
@@ -1,7 +1,27 @@
+const db = require('../../../db/db')
+const ActivityLog = require('../../activityLog/activityLogger')
 const ProcessingChainRepository = require('../repository/processingChainRepository')
 
+const updateChainProp = async (user, surveyId, processingChainUuid, key, value, client = db) => {
+  client.tx(t => {
```

Comment:
don't you need await here?

Reply:
it's not needed but it makes more evident that we are working with promises, I'll change it